lib/matplotlib/tests/test_ticker.py

```diff
@@ -1362,3 +1362,13 @@ def test_bad_locator_subs(sub):
     ll = mticker.LogLocator()
     with pytest.raises(ValueError):
         ll.subs(sub)
+
+
+@pytest.mark.parametrize('numticks', [1, 2, 3, 9])
+@pytest.mark.style('default')
+def test_small_range_loglocator(numticks):
+    ll = mticker.LogLocator()
+    ll.set_params(numticks=numticks)
+    for top in [5, 7, 9, 11, 15, 50, 100, 1000]:
+        ticks = ll.tick_values(.5, top)
+        assert (np.diff(np.log10(ll.tick_values(6, 150))) == 1).all()
```

Comment:
@tacaswell There seems to be a clear typo in the test here: you don't do anything with `ticks`, and the assert is the same on each loop iteration.  Do you remember what you intended to do?